architecture/README.md

```diff
@@ -33,127 +30,135 @@ types of roles. The TiDB community has the following organizations:
-    * May also review for more holistic issues
-* Expected to be responsive to review PR on time
-* Assigned test and fix bugs in subproject 
-* Have the rights to approve PR
-* Participate in the future seminar
+- Ensure the correctness of the code submitted to the SIG
+- Respond promptly to PRs
+- Participate in SIG meetings regularly
+- Approve code reviews with LGTM (Looks Good To Me)
+- Get the profile listed on the membership page of the corresponding SIG
 
 ### Committer
 
-Committers are developers of the projects and are elected from contributors.
-Currently.
+Committers come from Reviewers, and are recoganized by Maintainers and SIG Tech Leads for their outstanding contributions to the specific SIG.
 
-#### How to become a Committer?
+Committer is scoped to the specific SIG where the promotion applies. See `member-list.md` under the specific SIG for a list of Committers of the SIG.
 
-A new Committer must be nominated by any existing Maintainer or PMC member, and
-is generated through the voting of PMC members.
+#### How to become a Committer
 
 A Committer must have accomplished one or more of the following items:
 
-* Your commits have been approved and merged by Maintainers for over ten times.
-* You have implemented a complete feature.
-* You have fixed a vital bug.
+- Demonstrated a good sense of responsibility in PR reviews within the SIG. The review contributions must meet the amount or the difficulty criteria as defined in the charter of the specific SIG.
+- Demonstrated deep understanding of the TiDB components specific to the SIG by contributing significantly as:
+    - Finished 2+ tasks of Medium difficulty or higher
+    - Fixed 1+ task of Hard difficulty
+- Nominated by the Tech Leads of the SIG and gained supermajority (2/3) votes from Maintainers
 
-#### Responsibilities and Privileges
+#### Responsibilities and privileges
 
-* Recommend a new Organizer Committee Member
-* The profile can be listed on the Committer page
-* Participate in offline discussions and the design process
+- Control overall code quality of projects under the SIG
+- Guide Contributors and Reviewers to contribute to the SIG continuously
+- Participate in Committer meetings regularly and have knowledge priority regarding project development
+- Participate in design discussions
+- Get the profile listed on the membership page of the corresponding SIG
 
-#### Definition of Inactive Committer and operations:
+### Maintainer
 
-If a Committer has been inactive for over three months, the Committer is an
-Inactive Committer.  Concrete inactive behaviors are as follows:
+Maintainers are the planners and designers of the TiDB project, with the authority to merge branches into the master. Maintainers come from Committers. They must demonstrate both good judgment and responsibility towards the sustained development and overall health of the TiDB project. See [TiDB Maintainers](./maintainer-list.md) for a list of TiDB Maintainers.
```

Comment:
the url for maintain list is broken,

Reply:
I think it has to be merged to be effective because the maintainer list file is in master, not this branch.